gammapy/scripts/jupyter.py

```diff
@@ -0,0 +1,149 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Command line tool to perform devel management actions on jupyter notebooks."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from black import format_str
+import click
+import logging
+import nbformat
+import subprocess
+import sys
+import testipynb
+import time
+from unittest import TestCase
+from ..extern.pathlib import Path
+
+
+log = logging.getLogger(__name__)
+
+
+@click.command(name='black')
+@click.pass_context
+def cli_jupyter_black(ctx):
+    """Format code cells with black."""
+
+    nblist = build_nblist(ctx)
+
+    for notebook in nblist:
+
+        try:
+            nb = nbformat.read(str(notebook), as_version=nbformat.NO_CONVERT)
+        except Exception as ex:
+            log.error('Error parsing file {}'.format(str(notebook)))
+            log.error(ex)
+            sys.exit()
+
+        # paint cells in black
+        for cellnumber, cell in enumerate(nb.cells):
+            fmt = nb.cells[cellnumber]['source']
+            if nb.cells[cellnumber]['cell_type'] == 'code':
+                try:
+                    semicolon = 0
+                    fmt = comment_magics(fmt)
+                    if fmt.endswith(';'):
+                        semicolon = 1
+                    fmt = format_str(src_contents=fmt,
+                                     line_length=79).rstrip()
+                    if semicolon:
+                        fmt += ';'
+                except Exception as ex:
+                    logging.info(ex)
+                fmt = fmt.replace("###-MAGIC COMMAND-", "")
+            nb.cells[cellnumber]['source'] = fmt
+
+        # write formatted notebook
+        nbformat.write(nb, str(notebook))
+
+        # inform
+        print('Jupyter notebook {} painted in black.'.format(str(notebook)))
```

Comment:
Does the mix of print statements and log messages have a logic to it?
Maybe change to `log` messages everywhere?

Reply:
done.